spec/lib/rotp/hotp_spec.rb

```diff
@@ -108,14 +110,29 @@
   end
 
   describe '#provisioning_uri' do
-    it 'accepts the account name' do
+    let(:hotp)    { ROTP::HOTP.new('a' * 32, name: "m@mdp.im", issuer: "Example.com") }
```

Comment:
intentional whitespace? 

```suggestion
    let(:hotp) { ROTP::HOTP.new('a' * 32, name: "m@mdp.im", issuer: "Example.com") }
```